### ISSUE TITLE
[Firefox] 3rd-filters seem to be outdated even after update.

### ISSUE THREAD

**Issue by kostiya** [uBlock-LLC/uBlock#1390]
I am using uBlock v0.9.4.0.
After I am pushing the "Update Now", "Purge all caches", "Apply Changes"  Even EasyList seems to be outdated when I am opening it from cache (chrome://ublock/content/assets/thirdparties/easylist-downloads.adblockplus.org/easylist.txt).
The same problem seems to exist on all 3rd-filters.

Here's the beginning of the easylist.txt file after the update:
[Adblock Plus 2.0]
! Checksum: It9aRNcBfRy36ksxNxYlSA
! Version: 201505052300
! Title: EasyList
! Last modified: 05 May 2015 23:00 UTC
! Expires: 4 days (update frequency)
! Homepage: https://easylist.adblockplus.org/
! Licence: https://easylist-downloads.adblockplus.org/COPYING
!
! Please report any unblocked adverts or problems
! in the forums (https://forums.lanik.us/)
! or via e-mail (easylist.subscription@gmail.com).
!
!-----------------------General advert blocking filters-----------------------!
! **\* easylist:easylist/easylist_general_block.txt ***

**Comment by gorhill** [comment]
Duplicate of https://github.com/chrisaljoudi/uBlock/issues/860#issuecomment-75438762?

**Comment by kostiya** [comment]
Ya, It's the same issue.
Sorry for bothering.
